truffle.js

```diff
@@ -18,14 +18,12 @@ module.exports = {
   networks: {
     development: {
       host: "localhost",
-      port: 8545,
-      gas: 6712388,
-      // gasPrice: 2000000000, // 1 gwei
-      network_id: "*"
+      port: 7545,
```

Comment:
We wouldn't include changes in this file to a PR back to makoto as it prescribes a particular local blockchain implementation.